jest/setup-jest.js

```diff
@@ -1 +1,37 @@
 global.ResizeObserver = require("resize-observer-polyfill");
+
+const ERROR_PATTERNS_WE_SHOULD_FIX_BUT_ALLOW = [];
+const WARNING_PATTERNS_WE_SHOULD_FIX_BUT_ALLOW = [];
+
```

Comment:
This was taken from React Spectrum. This ensure that the SSR errors that are only logged in the console "useLayoutEffect not supported on the server" are throwed and fail the tests

Reply:
Orbit already have https://github.com/gsoft-inc/sg-orbit/blob/master/jest/utils/errorHandling.tsx for this purpose.